midterm-2023-11-06/code/5090-teodor-inovski/index.html

```diff
@@ -8,30 +8,35 @@
     <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+
     <h1>Bands Information</h1>
+
+    <div id="filter" class="filter">
+      <label for="name">Name: </label>
```

Comment:
The for in the label should refer to an `id` attribute of the input, so in this case, the correct value is `name-filter`